content/en/docs/instrumentation/js/manual.md

```diff
@@ -250,73 +476,331 @@ const opentelemetry = require("@opentelemetry/api");
+  if(isNaN(rolls)) {
+    res.status(400).send("Request parameter 'rolls' is missing or not a number.")
+    return
+  }
+  res.send(JSON.stringify(rollTheDice(rolls, 1, 6)));
+});
+
+app.listen(PORT, () => {
+  console.log(`Listening for requests on http://localhost:${PORT}`);
+});
+
+{{< /tab >}}
+
+{{< /tabpane>}}
+<!-- prettier-ignore-end -->
+
+And second, in the _library file_ `dice.ts` (or `dice.js`):
+
+<!-- prettier-ignore-start -->
+
+{{< tabpane langEqualsHeader=true >}}
+
+{{< tab TypeScript >}}
+/*dice.ts*/
+import { trace } from "@opentelemetry/api";
+
+const tracer = trace.getTracer(
+  'dice-lib',
+);
+
+function rollOnce(min: number, max: number) {
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
+export function rollTheDice(rolls: number, min: number, max: number) {
+  const result: number[] = []
+  for(let i = 0; i < rolls; i++) {
+    result.push(rollOnce(min, max));
+  }
+  return result
+}
+
+{{< /tab >}}
+
+{{< tab JavaScript >}}
+/*dice.js*/
+const { trace } = require("@opentelemetry/api");
+
+const tracer = trace.getTracer(
+  'dice-lib',
+);
+
+function rollOnce(min, max) {
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
+function rollTheDice(rolls, min, max) {
+  const result = []
+  for (let i = 0; i < rolls; i++) {
+    result.push(rollOnce(min, max));
+  }
+  return result
+}
+
+module.exports = { rollTheDice }
+{{< /tab >}}
+
+{{< /tabpane>}}
+
+<!-- prettier-ignore-end -->
+
+### Create spans
+
+Now that you have [`Tracer`s](/docs/concepts/signals/traces/#tracer)
+initialized, you can create [`Span`s](/docs/concepts/signals/traces/#spans).
+
+The below code sample shows how to create an active span, which is the most
 common kind of span to create.
 
+<!-- prettier-ignore-start -->
+
+{{< tabpane langEqualsHeader=true >}}
+
+{{< tab TypeScript >}}
+import { trace, Span } from "@opentelemetry/api";
+
+/* ... */
+
+export function rollTheDice(rolls: number, min: number, max: number) {
+  // Create a span. A span must be closed.
+  return tracer.startActiveSpan('rollTheDice', (span: Span) => {
+    const result: number[] = []
+    for(let i = 0; i < rolls; i++) {
+      result.push(rollOnce(min, max));
+    }
+    // Be sure to end the span!
+    span.end();
+    return result
+  })
+}
+{{< /tab >}}
+
+{{< tab JavaScript >}}
+function rollTheDice(rolls, min, max) {
+  // Create a span. A span must be closed.
+  return tracer.startActiveSpan('rollTheDice', (span) => {
+    const result = []
+    for (let i = 0; i < rolls; i++) {
+      result.push(rollOnce(min, max));
+    }
+    // Be sure to end the span!
+    span.end();
+    return result
+  })
+}
+{{< /tab >}}
+
+{{< /tabpane>}}
+
+<!-- prettier-ignore-end -->
+
+If you followed the instructions using the
+[example application](#example-application) up to this point, you can copy the
+code above in your _library file_ `dice.ts` (or `dice.js`). You should now be
+able to see spans being emitted by your application:
```

Comment:
Is there any reason for readers to not have "followed the instructions ... up to this point"? Isn't that the basic assumption in the page, that by defaul the reader will be using he example app?

```suggestion
If you followed the instructions using the
[example app](#example-app) up to this point, you can copy the
code above in your library file `dice.ts` (or `dice.js`). You should now be
able to see spans emitted from your app.
```